src/field.cpp

```diff
@@ -1661,6 +1669,23 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         make_rubble( p );
                         break;
 
+                    case fd_fungicidal_gas:
+                        {
+                            dirty_transparency_cache = true;
+                            spread_gas( cur, p, curtype, 120, 10 );
```

Comment:
120 in the third parameter of `spread_gas` shouldn't do anything above 100. It's a percentage chance.